content/en/docs/tasks/traffic-management/egress/egress-gateway/test.sh

```diff
@@ -58,11 +58,17 @@ _verify_contains snip_egress_gateway_for_http_traffic_7 "HTTP/2 200"
 
 # Verify routing through gateway
 if [ "$GATEWAY_API" == "true" ]; then
-    _verify_contains snip_egress_gateway_for_http_traffic_10 "outbound|80||edition.cnn.com"
+    _verify_contains snip_egress_gateway_for_http_traffic_11 "outbound|80||edition.cnn.com"
 else
     _verify_contains snip_egress_gateway_for_http_traffic_8 "outbound|80||edition.cnn.com"
 fi
 
+if [ "$GATEWAY_API" == "true" ]; then
+    verify_contains snip_egress_gateway_for_http_traffic_13 "$snip_egress_gateway_for_http_traffic_13_out"
+else
+    verify_contains snip_egress_gateway_for_http_traffic_10 "$snip_egress_gateway_for_http_traffic_10_out"
+fi
+
```

Comment:
Small nit. You can merge this if check with the previous one ... i.e., put the first check after line 61 and the second one after line 63